appnexus/model.py

```diff
@@ -92,6 +92,9 @@ def find(cls, **kwargs):
 
     @classmethod
     def find_one(cls, **kwargs):
+        if "id" not in kwargs:
+            raise Exception("Should provide an id to retrieve objects from "
+                            "'{}' service.".format(cls.service_name))
```

Comment:
Indeed, nice catch! Although it would be much simpler to just change the function signature to `def find_one(cls, id, **kwargs)`. :wink: